src/quo2/foundations/colors.cljs

```diff
@@ -318,3 +318,17 @@
 (defn dark?
   []
   (theme/dark?))
+
+(def chat-colors-with-theme
+  {"#fa6565" {:light (get-in customization [:red 50])
```

Comment:
curious, the colors mechanism should be able to handle a  hex string so was this just not working for the theming or the color would not display at all?

Reply:
It didn't display at all as it expected a keyword version of the color, So I needed to do the mapping so that quo2 could process the dark/light versions of the color.